doc/tutorial/cora-gcn.md

```diff
@@ -0,0 +1,168 @@
+
+Due to the fact that graph data is not applicable to be divided into batches, we are not able to split the data and store in training dataset and test dataset respectively. Thus, we represent the entire graph through two tables: `Node Table` and `Edge Table`. (Anyone who wants to store graph data in the database can refer to this method.)
+
+Here are the column description of the `Node Table`:
+
+Column | Explain 
+-- | -- 
+id| Id for the node.
+name| Name for the node.
+features| Feature vector of the node represented in the string. An example would be "0,0,1".
+label | Label for the node.
+
+The following command can be used to construct the `Node Table`. 
+
+```sql
+CREATE TABLE cora.node (
+        id INT,
+        node_name TEXT,
+        features  TEXT,
+        label TEXT);
+```
+
+Here are the column description of the `Edge Table`:
+
+Column | Explain 
+-- | -- 
+id| Id for the edge.
+from_node_id| Id for the from node of the edge.
+to_node_id| Id for the to node of the edge.
+weight | Weight for the edge.
+
+The following command can be used to construct the `Edge Table`. 
+
+
+```sql
+CREATE TABLE cora.edge (
+        id INT,
+        from_node_id INT,
+        to_node_id  INT,
+        weight FLOAT);
+```
+
+You can have a quick peek of the data by running the following standard SQL statements.
+
+```sql
+%%sqlflow
+describe cora.node;
+describe cora.edge;
+```
+
+```sql
+%%sqlflow
+select * from cora.node limit 10;
+```
+
+## Train GCN on the Cora dataset
+
+Let's train a GCN model!
+
+### Load data from Cora
+You can load the data from database following a standard SQL command such as `SELECT * FROM cora.node`. However, since GCN is supposed to deal with graph data, you have to load both the `Node Table` and `Edge Table` at once.
+
+In order to do so, you need to use the `JOIN` command in SQL to select all the data from `Node Table` and `Edge Table`. The following command is used to load all the data for training GCN.
+
+```sql
+%%sqlflow
+SELECT cora.node.id, features, label as class, cora.edge.from_node_id, cora.edge.to_node_id FROM cora.node
+LEFT JOIN cora.edge ON (cora.node.id = cora.edge.from_node_id OR cora.node.id = cora.edge.to_node_id)
+ORDER BY cora.node.id
+```
+The `OR` statement in the command is used to select all the possible bidirectional edges from the dataset. Without this `OR` command, some of the edges will be missing and it is not possible to construct the entire graph.
+
+With the `COLUMN` clause provided, SQLFlow can handle the comma separated string `features` with command `COLUMN DENSE(features)`.
+
+The GCN in SQLFlow is able to build the entire graph automatically with input in the folloing order: `node.id`, `node.features`, `node.label`, `edge.from_node_id`, `edge.to_node_id`. Please make sure the order is correct in order to run GCN successfully.
+
+### Train GCN
+After loading the dataset, you would be able to train GCN with following command:
```

Comment:
How about list all supported parameters in a table, and given a description each.

Reply:
Sure, I'm working on it.